variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -84,6 +89,7 @@ protected AbstractVariant() {
         this.end = -1;
         this.reference = null;
         this.alternate = null;
+        this.mainId = null;
```

Comment:
Is not code introduced in this PR, but, why do we need the no arguments constructor?

Reply:
The only things that might use it that I can think of are the JPA and automatic mongo conversion. I'm ok testing whether we need them, but maybe now is not the best moment